purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1976,9 +1990,24 @@ class Purchases internal constructor(
         var logHandler: LogHandler
             @Synchronized get() = currentLogHandler
             @Synchronized set(value) {
+                verboseLogHandler = verboseLogHandlerToLogHandler(value)
                 currentLogHandler = value
```

Comment:
I am still setting `currentLogHandler` to `value` in case devs assume they are actually setting that value. For example:

```
val myLogHandler = MyLogHandler()
Purchases.logHandler = myLogHandler
if (myLogHandler == Purchases.logHandler) // this requires us to set the currentLogHandler when setting logHandler, not setting it would make this fail

```